src/plone/restapi/imaging.py

```diff
@@ -1,8 +1,17 @@
+from plone.restapi import PLONE5
 from zope.component import getMultiAdapter
 from zope.component import getUtility
 from zope.globalrequest import getRequest
 
 
+if PLONE5:
+    # BBB: In Plone 5.2, it is necessary to use the direction parameter.
+    scale_parameter = {"direction": "thumbnail"}
+else:
+    # In Plone 6.0+, we must use the mode parameter
+    scale_parameter = {"mode": "thumbnail"}
```

Comment:
While add it you may want to replace the deprecated `thumbnail` with `scale`, see 
https://6.docs.plone.org/classic-ui/images.html#scaling-mode

Reply:
@jensens Done.